deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dashboard_saturation_tests"
+version = "1.0.0"
+dependencies = [
+    "pandas>=2.2.2",
+    "setuptools>=72.1.0",
+    "pycodestyle>=2.12.1",
+    "pyyaml>=6.0.2"
+]
+requires-python = ">=3.8"
```

Comment:
```suggestion
requires-python = ">=3.10"
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/7b6d04e9738b6ecfa7b067089d145b2ee5dd4603